src/index.ts

```diff
@@ -287,54 +286,76 @@ const extension: JupyterFrontEndPlugin<void> = {
      * previewEnabled flag and hides or shows the GCS browser or DPMS explorer
      * as necessary.
      */
-    const onPreviewEnabledChanged = () => {
-      previewEnabled = settings.get('previewEnabled').composite as boolean;
-      if (!previewEnabled) {
+    const onPreviewEnabledChanged = async () => {
```

Comment:
This method name (and the comment above it) is clearly out of date since there is no `preview enabled`  concept anymore.

Reply:
changed name and comments